### PR TITLE
Adds `continuous` option.

### PR DESCRIPTION
**Pull request by coreyleelarson**
First off, thank you for building spinnies! I've been looking for something exactly like this for a while.

Currently, once all spinners have either succeeded or failed, the spinners object is cleared and they can no longer be updated. I've added a `continuous` option to allow for the spinners to run and be updated indefinitely until the user manually terminates the process, as I am building a development tool and my use-case calls for it.

### CODE REVIEW

**Comment by jcarpanelli** [comment]
Hello @coreyleelarson! I like the idea of having an option for not removing the spinners, but still trying to come up with some use cases. Could you please explain yours? :slightly_smiling_face:

**Comment by coreyleelarson** [comment]
I am building an automated webpack build tool and, additionally, an internal plugin similar to `friendly-errors-webpack-plugin`. It will display a spinner for each config file being processed. In watch mode, it needs to be able to update the current spinners, even after all have been completed, as the watcher runs indefinitely until process is terminated by the user.

I have actually halted work on this for now, as we have higher priority items to attend to, so I don't really have an example to show. When I return to it, this will be something I'll need.

File: index.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ class Spinnies {
       failColor: 'red',
       spinner: terminalSupportsUnicode() ? dots : dashes,
       disableSpins: false,
+      continuous: false,
```
Comment (jcarpanelli) [inline index.js]: What about changing `continuous` to something like: `keepSpinners`?

Comment (coreyleelarson) [reply index.js]: `keepSpinners` sounds good to me.

**Pull request closed without merging**